floating_ip_test.go

```diff
@@ -81,7 +81,15 @@ func TestFloatingIpListRangesPaginateAndFilter(t *testing.T) {
 
 	floatingIpApi := FloatingIpApi{}
 	ctx := context.Background()
-	response, err := floatingIpApi.ListRanges(ctx, 1, 10, []string{"SITE", "METRO"}, "AMS-01")
+	opts := FloatingIpListOptions{
+		PaginationOptions: PaginationOptions{
+			Limit:  Int(1),
+			Offset: Int(10),
+		},
+		Type:     String("SITE, METRO"),
```

Comment:
This looks like it should be a `[]string`

Reply:
Done